creme/neighbors/knn_classifier.py

```diff
@@ -0,0 +1,151 @@
+import typing
+
+from creme import base
+from creme.utils import dict2numpy
+from creme.utils.math import softmax
+
+from .base_neighbors import BaseNeighbors
+
+
+class KNNClassifier(BaseNeighbors, base.Classifier):
+    """ k-Nearest Neighbors classifier.
+
+    This non-parametric classification method keeps track of the last
+    `window_size` training samples. The predicted class-label for a
+    given query sample is obtained in two steps:
+
+    1. Find the closest `n_neighbors` to the query sample in the data window.
+    2. Aggregate the class-labels of the `n_neighbors` to define the predicted
+       class for the query sample.
+
+    Parameters
+    ----------
+    n_neighbors : The number of nearest neighbors to search for.
```

Comment:
The format should be:

```py
n_neighbors
    The number of nearest neighbors to search for.
```

And no need to add an empty line between parameters.

Reply:
Changed in commit [64f91a8](https://github.com/creme-ml/creme/pull/323/commits/64f91a803c1bd8871fdb4e01694f6d1be14d8ad8).